src/main/java/com/flansmod/client/model/ModelVehicle.java

```diff
@@ -26,6 +26,17 @@
 	public ModelRendererTurbo rightTrackWheelModels[] = new ModelRendererTurbo[0];	//These go with the tracks but rotate
 	public ModelRendererTurbo leftTrackWheelModels[] = new ModelRendererTurbo[0];
 	
+	public ModelRendererTurbo leftAnimTrackModel[][] = new ModelRendererTurbo[0][0];  //Unlimited frame track animations
+	public ModelRendererTurbo rightAnimTrackModel[][] = new ModelRendererTurbo[0][0];
+	
+	public ModelRendererTurbo rightAnimTrackModel1[] = new ModelRendererTurbo[0]; //3 frame track animation
```

Comment:
Why do we have an unlimited frame track animation and a 3 frame one? Can't I just use the unlimited one with 3 frames?